pkg/inject/sidecar.go

```diff
@@ -132,11 +133,12 @@ func container(jaeger *v1.Jaeger, dep *appsv1.Deployment) corev1.Container {
 	jgBinaryTrft := util.GetPort("--processor.jaeger-binary.server-host-port=", args, 6832)
 
 	if len(util.FindItem("--jaeger.tags=", args)) == 0 {
-		agentTags := fmt.Sprintf("%s=%s,%s=%s,%s=%s,%s=%s",
+		agentTags := fmt.Sprintf("%s=%s,%s=%s,%s=%s,%s=%s,%s=%s",
 			"cluster", "undefined", // this value isn't currently available
 			"deployment.name", dep.Name,
 			"pod.namespace", dep.Namespace,
 			"pod.name", fmt.Sprintf("${%s:}", envVarPodName),
+			"ip", fmt.Sprintf("${%s:}", envVarHostIP),
```

Comment:
So it's not lost in the other thread, this needs to go back to `host.ip`, as when a tracer library submits traces with a process tag `ip`, the resulting trace will end up with two such tags, rather than one suppressing the other as I had assumed.